src/strategies/brightid/index.ts

```diff
@@ -16,12 +20,13 @@ export async function strategy(
   snapshot
 ) {
   const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const registry = options.registry[0] == "v" ? official[options.registry] : options.registry;
   const response = await multicall(
-    network,
+    options.registry[0] == "v" ? "74" : network,
```

Comment:
Any idea why this fail? 
```
    TypeError: Cannot read properties of undefined (reading 'toLowerCase')

      14 |     const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
      15 |     const registry = options.registry[0] == "v" ? official[options.registry] : options.registry;
    > 16 |     const response = await (0, utils_1.multicall)(options.registry[0] == "v" ? "74" : network, provider, abi, addresses.map((address) => [
         |                            ^
      17 |         registry,
      18 |         'isVerifiedUser',
      19 |         [address]
```

Try running `yarn test --strategy=brightid --more=500` in your local and make sure it pass all tests